src/ChunkReader.php

```diff
@@ -42,7 +41,7 @@ public function __construct(Container $container)
      */
     public function read(WithChunkReading $import, Reader $reader, TemporaryFile $temporaryFile)
     {
-        if ($import instanceof WithEvents && isset($import->registerEvents()[BeforeImport::class])) {
+        if ($import instanceof WithEvents) {
```

Comment:
This check is already being done in `beforeImport` so this was only preventing global events from firing (which is a bug)